addons/assistance.py

```diff
@@ -686,6 +686,26 @@ def check_console(self, message, channel, consoles):
         embed.description = "A Community-maintained homebrew database"
         await self.bot.say("", embed=embed)
 
+    #Information about autoRCM
+    @commands.command()
+    @commands.cooldown(rate=1, per=30.0, type=commands.BucketType.channel)
+    async def autorcm(self):
+        """Warnings about AutoRCM"""
+        await self.simple_embed("""
+                                AutoRCM is a tool to preserve eFuse count and automcatically boot to RCM when the console is powered off. __Most people don't need this__, but can be also used for convienience.
+                                
+                                • When the console powers off from the normal OS if AutoRCM is installed, it will automatically reboot to RCM. \
+You would need to inject Hekate and select `Power off` from the menu to fully turn the console off.
+                                
+                                • You will depend on your SD Card to take advantage of CFW. If anything happens to it\
+or your card slot, your console will only be able to boot stock firmware. \
```

Comment:
This is the case regardless, I don't see how that has anything to do with autoRCM specifically.

Reply:
You can still use Hekate and boot into stock without your SD Card inserted, but you wouldn't be able to use CFW. For users from <4.1.0, this will mean the GC fuse could be tripped on their SD-less boot. It also generally is a warning that you can only ever boot stock, if your SD card isn't inserted.

It can be removed.

Reply:
Fair points. How about something like 

```
You will depend on your SD Card to take advantage of CFW. If anything happens to it
or your card slot, your console will only be able to boot stock firmware. **Warning: if you updated from a firmware lower than 4.1.0, booting stock firmware can and will update your game cartridge reader, becoming unusable if you downgrade below 4.0 in the future!**
```